pkg/services/export/object_store.go

```diff
@@ -170,6 +183,71 @@ func (e *objectStoreJob) start() {
 		e.status.Last = fmt.Sprintf("ITEM: %s", playlist.Uid)
 		e.broadcaster(e.status)
 	}
+
+	// TODO.. query lookup
+	orgIDs := []int64{1}
```

Comment:
The next PR will replace the `git` export system to write to object store... that already supports iterating all the orgs, so this was just a quick pass to make sure we could write the main org